tooling/build/amplify/create-app.js

```diff
@@ -40,7 +40,13 @@ const createApp = async (appName) => {
     environmentVariables: {
       NEXT_PUBLIC_ISOMER_NEXT_ENVIRONMENT: "staging",
     },
-    customRules: [{ source: "/<*>", target: "/404.html", status: "404" }],
+    customRules: [
+      {
```

Comment:
actually what is this script used for?

Reply:
Yup, this is to create the Amplify apps using the current workflow of GitHub repositories, will be deprecated once we fully shift everyone to prod infra.